documentation/index.md

```diff
@@ -13,7 +13,12 @@ such as "Quests" to maintain compatibility with existing Quests on your server.
 
 Please visit the page that fits your needs the most:
 
-[Im a server owner! <span style="font-size:25px">:man_office_worker:</span>](For-Owners.md){: .md-button .md-button--primary }              
-[Im a quester! <span style="font-size:25px">:memo:</span>](For-Questers.md){: .md-button .md-button--primary }              
-[I want to contribute! <span style="font-size:25px">:handshake:</span>](Contributing/Contributing.md){: .md-button .md-button--primary }              
+[Im a server owner! <span style="font-size:25px">:man_office_worker:</span>](For-Owners.md){: .md-button .md-button--primary }
+&nbsp;&nbsp;&nbsp;
+[Im a quester! <span style="font-size:25px">:memo:</span>](For-Questers.md){: .md-button .md-button--primary }
+&nbsp;&nbsp;&nbsp;
+[I want to contribute! <span style="font-size:25px">:handshake:</span>](Contributing/Contributing.md){: .md-button .md-button--primary }
+&nbsp;&nbsp;&nbsp;
 [Im a developer! <span style="font-size:25px">:man_technologist:</span>](Developer-Documentation/API.md){: .md-button .md-button--primary }
+&nbsp;&nbsp;&nbsp;
+[Sponser this project! <span style="font-size:25px">:moneybag:</span>](Sponsorships.md){: .md-button .md-button--primary }
```

Comment:
The page now looks like this:
![grafik](https://user-images.githubusercontent.com/40303883/123965466-c527e980-d9b4-11eb-8b76-7f76af00b9ae.png)

Reply:
not for me, we should find a better solution for that on all resolutions

Reply:
It isn't any better currently:
![Screenshot_20210630-183755.jpg](https://user-images.githubusercontent.com/8677475/123999146-7b4dfc00-d9d2-11eb-876a-8ea4e3c5d7e6.jpg)